src/main/java/dev/jvfs/EscapeCrystalConfig.java

```diff
@@ -111,11 +111,22 @@ default boolean autoTeleTimerWhenInactive() {
         return false;
     }
 
+    @ConfigItem(
+            keyName = "autoTeleTimerTicks",
+            name = "Show infobox time in ticks",
+            description = "If active, show the infobox timer in ticks",
+            position = 8,
+            section = autoTeleTimerSection
+    )
+    default boolean autoTeleTimerTicks() {
+        return true;
```

Comment:
Thanks for the PR!

This should probably be `false` as the default, just so it doesn't change everyone's current settings on the fly once the plugin updates in case they don't want the timer showing as ticks